cmd/analyze/main.go

```diff
@@ -3,25 +3,34 @@ package main
 import (
 	"context"
 	"flag"
+	"fmt"
 	"net/url"
 	"os"
+	"strings"
 
 	"github.com/ossf/package-analysis/internal/analysis"
 	"github.com/ossf/package-analysis/internal/log"
 	"github.com/ossf/package-analysis/internal/pkgecosystem"
 	"github.com/ossf/package-analysis/internal/resultstore"
 	"github.com/ossf/package-analysis/internal/sandbox"
+	"github.com/ossf/package-analysis/internal/staticanalysis"
+	"github.com/ossf/package-analysis/internal/utils"
 	"github.com/ossf/package-analysis/internal/worker"
 )
 
 var (
-	pkgName   = flag.String("package", "", "package name")
-	localPkg  = flag.String("local", "", "local package path")
-	ecosystem = flag.String("ecosystem", "", "ecosystem (npm, pypi, or rubygems)")
-	version   = flag.String("version", "", "version")
-	upload    = flag.String("upload", "", "bucket path for uploading results")
-	noPull    = flag.Bool("nopull", false, "disables pulling down sandbox images")
-	imageTag  = flag.String("image-tag", "", "set a image tag")
+	pkgName         = flag.String("package", "", "package name")
+	localPkg        = flag.String("local", "", "local package path")
+	ecosystem       = flag.String("ecosystem", "", "ecosystem (npm, pypi, or rubygems)")
+	version         = flag.String("version", "", "version")
+	noPull          = flag.Bool("nopull", false, "disables pulling down sandbox images")
+	dynamicUpload   = flag.String("upload", "", "bucket path for uploading dynamic analysis results")
+	dynamicImageTag = flag.String("image-tag", "", "set image tag for dynamic analysis")
```

Comment:
I wonder if it's simpler to have one consistent tag for both dynamic and static when it comes to releasing. Managing separate sets of tags could get confusing fast.